manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: SpaceTime.Sheas.Cealer
+PackageVersion: 1.1.1
+InstallerType: inno
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

Reply:
Validation-Unattended-Failed still after this change